docs/resources/templates/Design-Document-Template.md

```diff
@@ -0,0 +1,52 @@
+Design Title: 
+Authors: 
+Date: 
+
+* Table of Contents
+{:toc}
```

Comment:
Not sure if this `{:toc}` tag works here.

Reply:
Removed as it depends on the document format (google doc has its own way). So it's up to the user to create one if the table of contents is needed.